docs/sources/tempo/operations/troubleshooting/metrics-generator.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Troubleshoot Metrics Generator
+menuTitle: Metrics Generator
+description: Gain an understanding of how to debug metrics quality issues.
+weight: 500
+---
+
+If you are concerned with data quality issues in the metrics generator, we'd first recommend:
```

Comment:
Consider adding a first-leveling heading for the page. This heading should match the page title. Also, our other doc uses metrics-generator with a hyphen. Is the official name metrics-generator or metrics generator? (I think I've seen both used.)

```suggestion

# Troubleshoot metrics-generator

If you are concerned with data quality issues in the metrics-generator, we'd first recommend:
```

Reply:
> Is the official name metrics-generator or metrics generator?

I would say that it's "metrics generator" but it is often written `metrics-generator` for use in yaml/helm/jsonnet/config etc.